socha-sdk/src/framework/sc/shared/GameResult.kt

```diff
@@ -0,0 +1,34 @@
+package sc.shared
+
+import com.thoughtworks.xstream.annotations.XStreamAlias
+import com.thoughtworks.xstream.annotations.XStreamImplicit
+import sc.framework.plugins.Player
+import sc.protocol.responses.ProtocolMessage
+
+@XStreamAlias(value = "result")
+class GameResult(
+        val definition: ScoreDefinition,
+        @XStreamImplicit(itemFieldName = "score")
+        val scores: List<PlayerScore>,
+        @XStreamImplicit(itemFieldName = "winner")
+        val winners: List<Player>?
+): ProtocolMessage {
+    
+    override fun toString(): String {
+        val builder = StringBuilder("Winner: ").append(winners)
+        for ((playerIndex, score) in scores.withIndex()) {
+            builder.append("\n").append("Player ").append(playerIndex).append(": ")
```

Comment:
> `builder.append("\n").append("Player ")`...
Is there a reason not to combine this to `builder.append("\nPlayer ")`?

Reply:
just tidyness, not in terms of implementation